sophon/utils/status.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+
+import commands
+
+command = {
+    "cpu": "top -b -n 1",
+    "disk": "df -h ."
+}
+
+def get_index(result, word):
+    index = 0
+    for x in result:
+        index = index + 1
+        if x == word:
+            return index
+    return 0
+
+def get_host_status():
+    ret = {
+        "Status": "",
+        "CPU Load": "", 
+        "Memory Usage": "",
+        "Disk Usage": ""
+    }
+    cpu = 'ansible all -a "%s"' % (command["cpu"])
+    disk = 'ansible all -a "%s"' % (command["disk"])
+    cpu_result = commands.getstatusoutput(cpu)
+    disk_result = commands.getstatusoutput(disk)
```

Comment:
`command` deprecated since version 2.6: The commands module has been removed in Python 3. Use the subprocess module instead.
eg: `cpu_result = subprocess.Popen(cpu, shell=True, stdout=subprocess.PIPE).stdout.read()`